src/main.tsx

```diff
@@ -1,48 +1,43 @@
 import "./styles/main.scss";
 // watch: native intellisense and file-peek for aliases from jsconfig.json and with none-js files doesn't work: https://github.com/microsoft/TypeScript/issues/29334
-import { BrowserRouter } from "react-router-dom";
-
-import { Component, ErrorInfo /* , StrictMode */ } from "react";
+import { BrowserRouter, Navigate, Route, Routes } from "react-router-dom";
+import { Component /* , StrictMode */ } from "react";
 import ReactDOM from "react-dom/client";
 
-import apiEndpoints from "./api.endpoints";
-
-import App from "./app";
-
-interface Props {}
-interface State {}
-
-async function testFetch(): Promise<void> {
-  const data = await (await fetch(apiEndpoints.testMock)).json();
-  console.warn("fetched data", data);
-}
-
-class AppContainer extends Component<Props, State> {
-  // ["constructor"]: typeof AppContainer;
+import PATHS from "./utils/paths";
 
-  constructor(props: Props) {
-    super(props);
-    this.state = {};
-    // test class-dead-code
-    const goExclude = true;
-    if (!goExclude) {
-      console.warn("class-dead-code doesn't work", props);
-    }
-  }
-
-  componentDidMount(): void {
-    setTimeout(testFetch, 300);
-  }
-
-  componentDidCatch(error: Error, errorInfo: ErrorInfo): void {
-    console.error("got err", { error, errorInfo });
-  }
+import Page from "./elements/page/page";
+import Home from "./components/home/home";
+import ErrorBoundary from "./components/error/errorBoundary/errorBoundary";
+import ErrorPage from "./components/error/errorPage/errorPage";
+import DefaultLayout from "./elements/defaultLayout";
 
+class AppContainer extends Component {
   render() {
     return (
       // <StrictMode>
       <BrowserRouter>
-        <App />
+        <ErrorBoundary>
+          <Routes>
+            {/* Routes wrapped with default layout */}
+            <Route
+              path="/*"
```

Comment:
This main-route-wrapper looks redundant here. I'm not 100% sure but try to remove it